src/DB/projects.json

```diff
@@ -14,6 +14,12 @@
         "title": "Simon-Game",
         "description": "Simon is an game of short-term memory skill",
         "tech": ["HTML", "CSS", "JavaScript"]
+      },
+      {
+        "link": "https://sabhisharma-ise.github.io/Quiz-Game/",
```

Comment:
add github link 
```suggestion
        "link": "",
```

Reply:
@priyankarpal should i add the github repo link of the Quiz-Game